scripts/deployment/deployApiConsumer.js

```diff
@@ -19,7 +19,7 @@ async function deployApiConsumer(chainId) {
     let oracleAddress
 
     if (chainId == 31337) {
-        const linkTokenFactory = await ethers.getContractFactory("LinkToken")
+        const linkTokenFactory = await ethers.getContractFactory("MockLinkToken")
```

Comment:
Couldnt comment against line 7 directly as it's unchanged, but its importing from v0.4 which is no longer a valid path.  Since this is JS the compiler wont warn, but i think if you run `npm run deploy` as per the README this fails?  I think the import part may be: @chainlink/contracts
/abi/v0.8/MockLinkToken.json